lttoolbox/entry_token.cc

```diff
@@ -77,9 +77,17 @@ EntryToken::setSingleTransduction(list<int> const &pi, list<int> const &pd, doub
 }
 
 void
-EntryToken::setRegexp(wstring const &r)
+EntryToken::setRegexp(UString const &r)
 {
-  myregexp = r;
+  //myregexp = r;
+  myregexp = vector<int32_t>(r.begin(), r.end());
```

Comment:
`vector.assign()` exists.

Reply:
except that it's also typecasting from `UChar` to `int32_t`, which I just realized will split code points.

really I just need to figure out how to write converters between 16 and 32.